packages/near-membrane-base/src/__tests__/environment.spec.ts

```diff
@@ -20,6 +17,36 @@ describe('VirtualEnvironment', () => {
             new VirtualEnvironment();
         }).toThrow(/Missing VirtualEnvironmentOptions options bag/);
     });
+    it('forwards support { ... } as bit fields to init functions', () => {
+        // Ignoring "Property 'assertions' does not exist on type '{...}'."
+        // @ts-ignore
+        expect.assertions(2);
+
+        // @ts-ignore
+        const interceptor: typeof init = (
+            _name: string,
+            _shouldOrNotTrapMutation: boolean,
+            supportFlags: SupportFlagsField,
+            exportsCallback: HooksCallback
+        ) => {
+            expect(supportFlags).toBe(0b1);
```

Comment:
What is `0b1` (besides a sick star wars reference)

Reply:
That was me being cute with the bit field. Since `supportFlags` is a bit field, I figured it made sense to use a binary literal value as the comparison.

Reply:
Sure, I can do that ;)